Makefile

```diff
@@ -145,14 +145,14 @@ standalone_package: venv package
 
 venv:
 	# Create a virtual environment which can be used to run the build package.
-	virtualenv -p ${PYTHON_BIN} venv && \
+	python3 -m venv venv && \
```

Comment:
I got the following error message when I tried to run this command on my machine:
```
The virtual environment was not created successfully because ensurepip is not
available.  On Debian/Ubuntu systems, you need to install the python3-venv
package using the following command.

    apt-get install python3-venv
```
Installing the `python3-venv` package solved the problem. This way we should list this dependency in the main readme file: https://github.com/Ericsson/codechecker/blob/master/docs/README.md#linux

Reply:
Thanks for checking it, I've extended the readme.